packages/components/addon/components/hds/app-footer/status-link.js

```diff
@@ -124,7 +112,7 @@ export default class HdsAppFooterStatusLinkComponent extends Component {
     let classes = ['hds-app-footer__status-link'];
 
     // add a class based on the @status argument
```

Comment:
it's no longer based solely on the argument value, but more an internally determined value
```suggestion
    // add a class based on status if no statusIconColor is explicitly specified
```